File: beginner/javascript/1177.js
var input = require('fs').readFileSync('stdin', 'utf8');
var linhas = input.split('\n');

var i, n, aux = 0;
var v = [];

n = parseInt(linhas.shift());

for(i = 0; i < 1000; i++){
    v[i] = aux;
    aux++;
    if(aux == n){
        aux = 0;
    }
}

for(i = 0; i < 1000; i++){
    console.log(`N[${i}] = ${v[i]}`);
}

File: beginner/python/1177.py
aux = 0
v = []

n = int(input())

for i in range(0,1000):
    v.append(aux)
    aux+=1
    if aux == n:
        aux = 0
    

for i in range(0,1000):
    print("N[%d] = %d" % (i, v[i]))
